addons/binding/org.openhab.binding.mpower/src/main/java/org/openhab/binding/mpower/internal/connector/SSHExecutor.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.mpower.internal.connector;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.jcraft.jsch.ChannelExec;
+import com.jcraft.jsch.JSchException;
+import com.jcraft.jsch.Session;
+
+/**
+ * handles raw SSH command execution
+ *
+ * @author Marko Donke - Initial contribution
+ *
+ */
+public class SSHExecutor {
+    private Session session;
+    private final static Logger logger = LoggerFactory.getLogger(SSHExecutor.class);
```

Comment:
If you make it static, you must here because you use it from a static method, also make it in CAPS to follow the naming conventions, maybe add a comment on top why it is static.